files/build_templates/docker_image_ctl.j2

```diff
@@ -175,19 +175,16 @@ start() {
 {%- endif %}
 {%- if sonic_asic_platform == "mellanox" %}
 {%- if docker_container_name == "syncd" %}
-        -e SX_SNIFFER_ENABLE \
```

Comment:
>SX_SNIFFER_ENABLE [](start = 11, length = 17)

why these `-e` removed?

Reply:
these are some leftover unused stuff and should be removed.  previously we intend to pass environment variables to Mellanox SDK to enable/disable some sniffer functions. But eventually we do it in another way, these '-e' are not used anymore.

Reply:
the design doc is here: https://github.com/Azure/SONiC/wiki/Mellanox-SDK-and-PRM-Sniffer-Utility-CLI-Design-for-SONiC#22-manipulate-related-env-variable-for-sniffer   we manipulate supervisord conf instead of using these "-e" options.